bindings/bindingtester/cbatrace/cbatracefdb_c.g.cpp

```diff
@@ -0,0 +1,674 @@
+// ////////////////////////////////////////////////////////////////////////////////////// //
```

Comment:
does this need to be checked in? I would rather have cmake generate it than have it in the repository